apache/superset
# feat: Programmatically open "more filters" dropdown in Horizontal Filter Bar

### SUMMARY
This PR adds the ability to open the "more filters" dropdown in the horizontal filter bar while refactoring the code to better differentiate between focus and hover actions on native filters.

### AFTER

https://user-images.githubusercontent.com/60598000/204836413-190dc840-a2b3-4c1f-8385-86e31837947f.mp4

### TESTING INSTRUCTIONS
1. Enable HORIZONTAL_FILTER_BAR feature flag
2. Open a dashboard
3. Set the filter bar to horizontal
4. Click on a filter that is currently within the "more filters" dropdown
5. The dropdown should open and focus the filter

### ADDITIONAL INFORMATION
